src/Tribe/Values/Value_Interface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tribe\Values;
+
+interface Value_Interface {
```

Comment:
`Camel_Snake_Case` 
Also docblocks for all methods. Set the Docblocks here, and `@inheritDoc` on all others